### ISSUE TITLE
Processing stdin

### ISSUE THREAD

**Issue by nunojpg** [CroatiaControlLtd/asterix#91]
Hi!

My understanding of usage is that the following two lines should be equivalent:

```
$ asterix -f file.asterix
$ cat file.asterix | asterix
```

But there are two problems here, one is that asterix always expects some arguments, otherwise it prints the help. This can be workaround with a dummy argument. The other is that it still fails to process the stdin input, and loops forever with error messages like:

```
ERROR:   May 17 11:01:21 2019 (converterengine.cxx) ReadPacket() failed.
ERROR:   May 17 11:01:21 2019 (asterixrawsubformat.cxx) Not enough data for packet! Size = 142, left = 0.
```

Let me know if this is not a issue and I'm just making totally wrong assumptions.

Thanks!

**Comment by dsalantic** [comment]
You are right, this is a bug. I'll fix that.

**Comment by dsalantic** [comment]
It's fixed now in version 2.5.0 (python_v0.5.10)
